academy/ibc-dev/ibcapp_checkers.md

```diff
@@ -0,0 +1,525 @@
+# Extend the Checkers Game With a Leaderboard
+
+Go to your checkers folder and make sure that you checked out of the [v1-cosmjs-elements](https://github.com/cosmos/b9-checkers-academy-draft/tree/v1-cosmjs-elements) tag.
```

Comment:
This is a commit from the earlier checkers project with ignite cli v0.17, which gives a notification to migrate from Ignite CLI when attempting next scaffolding step with Ignite CLI v0.22

Reply:
Replace with branch `cosmjs-elements` which is the current version.

Reply:
Indeed I used the cosmjs-elements branch for the test I did at https://github.com/b9lab/cosmos-ibc-docker/tree/ao-modular/modular, but I wanted to have a tag like the one you have for v1. So I have been keeping in mind to replace this link as soon as you have a v2 tag, which I assumed would be included for the next cohort.

Reply:
My branches are still subject to change a little, which is why I'd rather only create the v2 tags when working on v3 😬.
FYI, if you look at the commits, you see that each branch has 2 commits. It was a long and arduous process with a lot of squashing and reordering, but it makes for a cleaner "history". 
I want to keep it this way, so the content of the branch may change although the commits' dates and positions don't.